Node.gitignore

```diff
@@ -30,6 +30,7 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages
+typings
```

Comment:
Can you document this and indicate it's only relevant for Typescript pre-v2 projects?